package/gluon-autoupdater/luasrc/lib/gluon/upgrade/500-autoupdater

```diff
@@ -41,11 +41,16 @@ local seed1, seed2 = urandom:read(2):byte(1, 2)
 math.randomseed(seed1*0x100 + seed2)
 urandom:close()
 
--- Perform updates at a random time between 04:00 and 05:00, and once an hour
--- a fallback update (used after the regular updates haven't worked for
--- (priority+1) days after a firmware release, for example because the node
--- is always offline at night)
-local minute = math.random(0, 59)
+-- Perform updates at a random time between 04:00 and 05:00 in case no specific
+-- time is specified (gluon.autoupdater.minute), and once an hour a fallback
+-- update (used after the regular updates haven't worked for (priority+1) days
+-- after a firmware release, for example because the node is always offline at
+-- night)
+if not uci:get('gluon', 'autoupdater') then
+	uci:section('gluon', 'autoupdater', 'autoupdater')
```

Comment:
do i understand correctly, that this adds a new empty autoupdater section in the config, if there is no such section, yet?

Reply:
why did you create a new section `gluon.autoupdater.minute`? and not just store it in `autoupdater.minute`?

Reply:
@rubo77 See NeoRaiders [comment](https://github.com/freifunk-gluon/gluon/pull/1266#discussion_r158950135) above.